docs/docs/30-administration/15-agent-config.md

```diff
@@ -153,4 +153,29 @@ Configures if the gRPC server certificate should be verified, only valid when `W
 ### `WOODPECKER_BACKEND`
 > Default: `auto-detect`
 
-Configures the backend engine to run pipelines on. Possible values are `auto-detect`, `docker`, or `local`.
+Configures the backend engine to run pipelines on. Possible values are `auto-detect`, `docker`, `local` or `ssh`.
+
+### `WOODPECKER_SSH_HOST`
```

Comment:
I think using HOST is misleading. 

Since it also seems to support "ssh.example.org:2234", it would be more than the host, as it include the port. So either the post should be a separate variable, or it should reuse the same nomenclature as goph, eg, use address (or addr).